specifyweb/frontend/js_src/lib/components/Molecules/ResourceLink.tsx

```diff
@@ -73,18 +74,22 @@ export function ResourceLink<COMPONENT extends typeof Link['Icon']>({
       <AnyComponent {...allProps} />
       {isOpen && (
         <IsNotReadOnly.Provider value>
-          <LazyResourceView
-            dialog="modal"
-            onAdd={undefined}
-            onSaved={(): void => {
-              resourceView.onSaved?.();
-              handleClose();
-            }}
-            {...resourceView}
-            isDependent={false}
-            isSubForm={false}
-            resource={resource}
-            onClose={handleClosed}
+          <Async
```

Comment:
(optional) since you are not using Async's set title feature, and it wasn't adapted to handle receiving JSX, I see less value in using Async here instead of `<React.Suspense fallback={<LoadingScreen />}>`

up to you

Reply:
Yeah, I think it took "sharing code" too far here (see https://github.com/specify/specify7/pull/4331/commits/adbd22b8a809c729af1b866f9dfe7350f3b892e3), where it was just Suspense and I used Async afterwards

Reply:
Will revert to that commit, doesn't seem worth here